ci/ray_ci/automation/check_nightly_ray_commit.py

```diff
@@ -0,0 +1,59 @@
+import click
+import sys
+
+from ci.ray_ci.utils import logger
+from ci.ray_ci.automation.docker_tags_lib import (
+    list_image_tags,
+    get_ray_commit,
+    pull_image,
+    remove_image,
+)
+from ci.ray_ci.docker_container import (
+    PLATFORMS_RAY,
+    PLATFORMS_RAY_ML,
+    PYTHON_VERSIONS_RAY,
+    PYTHON_VERSIONS_RAY_ML,
+    ARCHITECTURES_RAY,
+    ARCHITECTURES_RAY_ML,
+    RayType,
+)
+
+
+@click.command()
+@click.option("--ray_type", required=True, type=RayType)
+@click.option("--expected_commit", required=True, type=str)
+def main(ray_type, expected_commit):
```

Comment:
could you move the content of this function into the `py_library` and add some unit tests for the logic?

Reply:
done